data/channels.csv

```diff
@@ -18798,629 +18791,626 @@ MATV.mg,MATV,,,,MG,,Antananarivo,c/MG,fra,general,FALSE,1995-05-01,,,http://matv
 MCAETTV.us,MCAET TV,,,,US,US-CA,Salinas,s/US-CA,eng,,FALSE,,,,https://mcaet.org/,https://i.imgur.com/Yc94310.png
 MCAMGovernmentChannel.us,MCAM Government Channel,,,,US,US-MN,Moorhead,s/US-MN,eng,legislative,FALSE,,,,https://moorheadaccess.com/government-channel.html,https://i.imgur.com/fWWwLxX.png
 MCAMPublicAccessChannel.us,MCAM Public Access Channel,,,,US,US-MN,Moorhead,s/US-MN,eng,general,FALSE,,,,https://moorheadaccess.com/public-access-channel.html,https://i.imgur.com/fWWwLxX.png
 MCATChannel189.us,MCAT Channel 189,,,,US,US-MT,Missoula,s/US-MT,eng,general,FALSE,,,,https://mcat.org/watch/,https://i.imgur.com/1TTFbZh.png
 MCATChannel190.us,MCAT Channel 190,,,,US,US-MT,Missoula,s/US-MT,eng,legislative,FALSE,,,,https://mcat.org/watch/,https://i.imgur.com/1TTFbZh.png
 MCC.id,MCC,,,,ID,,,c/ID,ind,,FALSE,,,,https://maxstream.tv/,https://i.imgur.com/aACrLKU.png
 MCCAM.us,MCCAM,,,,US,US-MA,Middleborough,s/US-MA,eng,general,FALSE,,,,https://mccam02346.com/,https://i.imgur.com/2LPmk2u.png
-MCEU.tr,MC EU,,,,TR,,,c/TR,tur,,FALSE,,,,http://mceu.tv/,https://i.imgur.com/taZpcZt.png
+MCEU.de,MC EU,,,Itep Pictures GmbH,DE,,Oberthausen,c/DE;c/TR,tur,entertainment,FALSE,,,,http://mceu.tv/,https://i.imgur.com/taZpcZt.png
 MChannel.id,M Channel,,,,ID,,,c/ID,ind,,FALSE,,,,,https://i.imgur.com/pZ8acgI.png
 MChannel.mz,M Channel,,,,MZ,,,c/MZ,por,,FALSE,,,,,https://www.lyngsat.com/logo/tv/mm/mchannel-mz.png
-MChannel.th,M Channel,,,,TH,,,c/TH,tha,,FALSE,,,,,https://i.imgur.com/vmETgay.png
+MChannel.th,M Channel,,,Major Katana Broadcasting Co Ltd,TH,,Bangkok,c/TH,tha,,FALSE,,,,,https://i.imgur.com/vmETgay.png
 McKinneyTV.us,McKinney TV,,,,US,US-TX,McKinney,s/US-TX,eng,legislative,FALSE,,,,https://www.mckinneytexas.org/229/Video-Production,https://i.imgur.com/oLpPaVE.png
 McLeodsDaughters.uk,McLeod's Daughters,,Pluto TV,,UK,,,c/UK,eng,series,FALSE,,,,https://pluto.tv/en/live-tv/mcleods-daughters-gb,https://i.imgur.com/q4BBDzw.png
-MCM.fr,MCM,,,,FR,,,c/FR,fra,music,FALSE,1989-07-01,,,http://www.mcm.fr/,https://i.imgur.com/F5WNCp0.png
-MCMPop.fr,MCM Pop,MCM2,,MCM Group,FR,,,c/PT,fra,music,FALSE,2003-11-28,2014-10-01,,http://www.mcm.fr/mcm-pop/,https://i.imgur.com/j31lBL7.png
-MCMTop.fr,MCM Top,,,,FR,,,c/FR,fra,music,FALSE,2003-11-28,,,http://www.mcm.fr/mcm-top/,https://i.imgur.com/EVtWQLd.png
-MCMTop.ru,MCM Top,MCM Top,,,RU,,,c/RU,fra,music,FALSE,2015-03-01,,,https://universaltv.ru/channel/MCM-TOP,https://i.imgur.com/EVtWQLd.png
+MCM.fr,MCM,,,Groupe M6,FR,,Neuilly-sur-Seine,c/FR,fra,music,FALSE,1989-07-01,,,http://www.mcm.fr/,https://i.imgur.com/F5WNCp0.png
+MCMPop.fr,MCM Pop,MCM2,,MCM Group,FR,,Neuilly-sur-Seine,c/PT,fra,music,FALSE,2003-11-28,2014-10-01,,http://www.mcm.fr/mcm-pop/,https://i.imgur.com/j31lBL7.png
+MCMTop.fr,MCM Top,,,Groupe M6,FR,,Neuilly-sur-Seine,c/FR,fra,music,FALSE,2003-11-28,,,http://www.mcm.fr/mcm-top/,https://i.imgur.com/EVtWQLd.png

```

Comment:
Available in Belgium, Brazil, Monaco, Switzerland, Portugal as well.

Reply:
Got it: thank you